src/cheatsheet.html

```diff
@@ -23,9 +23,9 @@ <h2 class="page-header">Every Font Awesome {{ site.fontawesome.version }} Icon,
     {% assign sorted_icons = icons | expand_aliases | sort_by:'class' %}
 
     {% for icon in sorted_icons %}
-    <div class="col-md-4 col-sm-6 col-lg-3">
+    <div class="cheatsheet-icon">
       {% if icon.created >= site.fontawesome.major_version %}<small class="text-muted pull-right">{{ icon.created }}</small>{% endif %}
-      <i class="fa fa-fw" aria-hidden="true" title="Copy to use {{ icon.class }}">&#x{{ icon.unicode }}</i>
+      <i class="fa fa-fw" aria-hidden="true" title="Copy to use {{ icon.class }}">&#x{{ icon.unicode }};</i>
```

Comment:
@tagliala I don't think the semicolon is needed here. Its needed a bit below (line  31), but the current site renders the icons just fine without it.